lib/query_builder/schema.ex

```diff
@@ -51,6 +51,10 @@ defmodule EctoShorts.QueryBuilder.Schema do
     end)
   end
 
+  defp create_schema_field_filter(query, filter_field, nil) do
```

Comment:
this will actually break functionality in a few applications, how to filter nil would actually be `%{field: %{==: nil}}`